### PR TITLE
Updated Wimp.com extractor to work with newer jPlayer code using data…

### PR DESCRIPTION
**Pull request by stuporglue**
…-dash values in the video div tag

## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x ] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x ] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x ] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

The Wimp.com extractor in master is broken because Wimp.com has changed how it uploads videos and the code on its site.

The old version had the jwplayer values in JavaScript, the new code stores the values in HTML5 data attributes of the video player div. 

It seems that they have also re-encoded some videos and updated titles and descriptions on some older videos. 

This pull request: 

1) Fixes the existing tests to work again. 
2) Adds detection of the new way they're embedding videos.
3) Adds a new test for the new embed method case.
4) Falls back on the old/existing method of downloading videos if the new method is not detected. 

I've tried to follow the youtube-dl coding standards as I understand them and the flake8 doesn't complain. 
--
Michael / stuporglue

### CODE REVIEW

File: youtube_dl/extractor/wimp.py [hunk 1]
```diff
@@ -45,13 +51,23 @@ def _real_extract(self, url):
                 'ie_key': YoutubeIE.ie_key(),
             }
 
-        info_dict = self._extract_jwplayer_data(
-            webpage, video_id, require_title=False)
+        url = self._search_regex(r'class=\'oyt-container .*? data-src=\'(.*?)\'', webpage, 'Video URL', default=None, fatal=False)
```
Comment (dstftw) [inline youtube_dl/extractor/wimp.py]: Do not shadow `url` variable.  `fatal` has no effect when `default` is provided.

File: youtube_dl/extractor/wimp.py [hunk 2]
```diff
@@ -45,13 +51,23 @@ def _real_extract(self, url):
...
+
+        if url:
+            info_dict = {
+                'id': video_id,
+                'title': self._og_search_title(webpage),
+                'description': self._og_search_description(webpage),
```
Comment (dstftw) [inline youtube_dl/extractor/wimp.py]: Code duplication at 58-60 and 68-70.

**Comment by stuporglue** [comment]
Hi @dstftw, 

GitHub says 

> dstftw requested changes 5 hours ago

Do I need to do anything with those changes? 

I can't tell if GitHub is telling me that you're requesting that I change my pull request, or if you've already made changes and someone in the youtube-dl project needs to accept them. 

Since you're marked as _collaborator_ I'm guessing that I don't have to do anything else to accept your changes, but please let me know if I do. 

Thanks,
Michael

**Comment by dstftw** [comment]
Not sure what do you mean. What you should is to make changes to your code to meet the code review, squash commits and then push them to this branch.

**Comment by dstftw** [comment]
No longer actual.

**Pull request closed without merging**